app/routes/helpers/merge-claim-expenses-with-submitted-responses.js

```diff
@@ -12,15 +12,14 @@ module.exports = function (oldData, nonPersistedData) {
   oldData.forEach(function (expense) {
     var postedClaimExpenseResponse = claimExpensesById[expense.ClaimExpenseId.toString()]
     expense.Status = postedClaimExpenseResponse.status
-    if (expense.Status === claimDecisionEnum.APPROVED_DIFF_AMOUNT) {
+    if (expense.Status === claimDecisionEnum.APPROVED_DIFF_AMOUNT || expense.Status === claimDecisionEnum.MANUALLY_PROCESSED) {
       expense.ApprovedCost = postedClaimExpenseResponse.approvedCost
       if (!Validator.isCurrency(postedClaimExpenseResponse.approvedCost) || !Validator.isGreaterThanZero(postedClaimExpenseResponse.approvedCost) ||
         !Validator.isLessThanMaximumDifferentApprovedAmount(postedClaimExpenseResponse.approvedCost)) {
         expense.Error = true
       }
     }
   })
-
```

Comment:
leave this blank line in as per https://github.com/airbnb/javascript#whitespace--after-blocks